providers/apis/defi/raydium/price_fetcher.go

```diff
@@ -70,7 +70,14 @@ func NewAPIPriceFetcher(
 				client,
 			))
 		} else {
-			config.URL = config.Endpoints[0].URL
+			client, err := solanaClientFromEndpoint(config.Endpoints[0])
```

Comment:
This prevents people from using URL. Should probably fail validation if they only provide a URL then?